recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+        self._cmake.definitions["CRYPTO_LIB"] = self.options.crypto_lib
+        self._cmake.definitions["MSIX_PACK"] = self.options.pack
+        self._cmake.definitions["MSIX_SAMPLES"] = False
+        self._cmake.definitions["MSIX_TESTS"] = False
+        self._cmake.definitions["SKIP_BUNDLES"] = self.options.skip_bundles
+        self._cmake.definitions["USE_EXTERNAL_ZLIB"] = self.options.use_external_zlib
+        self._cmake.definitions["USE_SHARED_ZLIB"] = self.options["zlib"].shared
+        self._cmake.definitions["USE_VALIDATION_PARSER"] = self.options.use_validation_parser
+        self._cmake.definitions["XML_PARSER"] = self.options.xml_parser
+        self._cmake.configure()
+        return self._cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == "Linux" and (self.settings.compiler != "clang"):
+            raise ConanInvalidConfiguration("Only clang is supported on Linux")
+        if self.settings.os == "Linux" and (self.options.crypto_lib == "crypt32"):
+            raise ConanInvalidConfiguration("crypt32 is not available on Linux")
+        if self.settings.os == "Linux" and self.options.shared:
+            self.options.fPIC = True
+        if self.settings.os != "Android" and self.options.xml_parser == "javaxml":
+            raise ConanInvalidConfiguration("javaxml is supported only for Android")
+        if self.settings.os != "Macos" and self.options.xml_parser == "applexml":
+            raise ConanInvalidConfiguration("applexml is supported only for MacOS")
+        if self.settings.os != "Windows" and self.options.xml_parser == "msxml6":
+            raise ConanInvalidConfiguration("msxml6 is supported only for Windows")
+        if self.options.pack:
+            if self.settings.os == "Macos":
+                if not self.options.use_external_zlib:
+                    raise ConanInvalidConfiguration("Using libCompression APIs and packaging features is not supported")
+                if self.options.xml_parser and (self.options.xml_parser != "xerces"):
+                    raise ConanInvalidConfiguration("Xerces is the only supported parser for MacOS pack")
+            if not self.options.use_validation_parser:
+                raise ConanInvalidConfiguration("Packaging requires validation parser")
+        if self.options.xml_parser == "xerces":
+            self.options["xerces-c"].char_type = "char16_t"
+            self.options["xerces-c"].shared = False
+        tools.check_min_cppstd(self, "14")
+
+    def requirements(self):
+        if self.settings.os == "Linux" and not self.options.skip_bundles:
+            self.requires("icu/68.2")
+        if self.settings.os != "Windows" and not self.options.crypto_lib:
+            self.requires("openssl/1.0.2t")
+        if self.options.crypto_lib == "openssl":
+            self.requires("openssl/1.0.2t")
```

Comment:
Whey you apply https://github.com/conan-io/conan-center-index/pull/6475/files#r675915208, this can be simplified to:
```suggestion
        if self.options.crypto_lib == "openssl":
            self.requires("openssl/1.0.2t")
```

Reply:
[Fixed](https://github.com/conan-io/conan-center-index/pull/6475/commits/0e77bd7ec9eab25036be96cd42fb99bc5297c326)